vhdbuilder/packer/install-dependencies.sh

```diff
@@ -479,12 +475,16 @@ K8S_VERSIONS="
 1.18.6-hotfix.20200723.1
 1.18.8-hotfix.20200924
 1.18.10
+1.18.10-hotfix.20210118
```

Comment:
For 20210118 hotfix, we also have `v1.18.14-hotfix.20210118` and `v1.19.3-hotfix.20210118` etc, why don't we also include the hotfixes here?

Reply:
1.18.14 hotfix is added below. 1.19.3 will be removed from supported version, since we'll support 1.19.6 and 1.19.7